packages/frontend/src/trpc/query-client.ts

```diff
@@ -1,6 +1,5 @@
 import { UnixTime } from '@l2beat/shared-pure'
 import { QueryClient, defaultShouldDehydrateQuery } from '@tanstack/react-query'
-import SuperJSON from 'superjson'
 
 export const createQueryClient = () =>
```

Comment:
Note for future changes. Do we even use SSR for query-client?
If yes then we're 100% not sending the data to the client.
If no then this config is probably wrong.